packages/api-reference/src/components/Content/ClientLibraries/ClientSelector.vue

```diff
@@ -78,29 +78,29 @@ const checkIfClientIsFeatured = (client: HttpClientState) =>
     <button
       v-for="client in featuredClients"
       :key="client.clientKey"
+      aria-hidden="true"
       class="client-libraries rendered-code-sdks"
       :class="{
         'client-libraries__active': isSelectedClient(client),
       }"
+      tabindex="-1"
```

Comment:
this prevent the button from being outlined on tab and selected, is that what we want?